Libs/MRML/Core/vtkITKTransformConverter.h

```diff
@@ -23,6 +23,8 @@
 #include <vtkPoints.h>
 #include <vtkThinPlateSplineTransform.h>
 #include <vtkTransform.h>
+#include <vtkCollection.h>
```

Comment:
These are probably not actually needed (due to how usually other headers are included when using this converter), but if you want to add them it's fine, just keep the alphabetical order.

Reply:
I added them to resolve compile errors. I now ordered them alphabetically.